script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -227,7 +227,8 @@ Entering a person's room as you please and straddling them while they're asleep,
 [sha:8636506e61a97ce9b587252888fb651a82420a46]{
 -- Page 15, Offset 24303.
 -- 人を待たせておいて自分はずっと眠っているなんて、いったいどういうつもりなの？」
-Where do you get off, keeping people waiting while you yourself are happily snoozing away?"
+// Not sure what the original line was going for, exactly
```

Comment:
"Where do you get off" = how dare you. I'd revert.

Reply:
Maybe it's just me, but I've never heard this term before now. Am I the only one?

Reply:
I don't think it's a Britishism, check with Ben.

Reply:
commented on discord, but not a britishism. Common expression